### PR TITLE
Remove "interactive" from podspec

### PR DESCRIPTION
**Pull request by designatednerd**
Also bump version so we can submit an update to CocoaPods.

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/ios-developers "code review", anyone?

**Comment by brockboland** [comment]
LGTM

File: Vokoder.podspec [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  s.version          = "1.1.1"
-  s.summary          = "VOKAL's Core Data Manager"
-  s.homepage         = "https://github.com/vokalinteractive/Vokoder"
+  s.version          = "1.1.2"
+  s.summary          = "Vokal's Core Data Manager"
+  s.homepage         = "https://github.com/vokal/Vokoder"
   s.license          = { :type => "MIT", :file => "LICENSE"}
-  s.author           = { "VOKAL Interactive" => "hello@vokalinteractive.com" }
-  s.source           = { :git => "https://github.com/vokalinteractive/Vokoder.git", :tag => s.version.to_s }
+  s.author           = { "Vokal" => "hello@vokal.io" }
```
Comment (chillpop) [inline Vokoder.podspec]: has email on the new domain been set up yet?

**Comment by chillpop** [comment]
How about changing the copyright name in the LICENSE and source header comments? Or is that under a different issue?

**Comment by designatednerd** [comment]
@chillpop Ooh, good call

File: Vokoder.podspec (hunk 1 above)
Comment (vokal-isaac) [inline Vokoder.podspec]: Apparently—I just sent a message from a non-Google-based email address to my email at vokal.io and received it.

Comment (designatednerd) [inline Vokoder.podspec]: Yes, I've been getting it since before xmas.

**Comment by designatednerd** [comment]
@chillpop Better?

**Comment by chillpop** [comment]
LGTM.

I noticed that there is no copyright in the actual Pod source files.  Perhaps we should add one?

**Comment by designatednerd** [comment]
Yeah it's wildly inconsistent throughout. Do you want me to address that here or just file an issue for when we have time?

**Comment by chillpop** [comment]
@designatednerd, I'd say add another issue unless you're really bored.

**Comment by designatednerd** [comment]
Less bored than procrastinating, so I'll close this and file an issue.

**Pull request merged**